osu.Game/Screens/SelectV2/SongSelect.cs

```diff
@@ -455,7 +455,7 @@ protected override void LogoArriving(OsuLogo logo, bool resuming)
         {
             base.LogoArriving(logo, resuming);
 
-            if (logo.Alpha > 0.8f)
+            if (logo.Alpha > 0.8f && resuming)
```

Comment:
We don't want to animate the tracking start when entering song select because main menu starts with the logo off the side of the screen (to the left).